phpunit/class-rest-batch-controller-test.php

```diff
@@ -0,0 +1,232 @@
+<?php
+/**
+ * REST API: WP_REST_Batch_Controller tests.
+ *
+ * @package Gutenberg
+ */
+
+/**
+ * Tests for the batch controller.
+ */
+class REST_Batch_Controller_Test extends WP_Test_REST_TestCase {
```

Comment:
It would be good to test if `allow_batch` opt-in works - I don't think any test case actually sets it at the moment.

Reply:
Isn't that covered in `test_batch_requires_allow_batch_opt_in`?

Reply:
It seems like it tests the case where `allow_batch` is not set, but I don't see any test that actually sets `allow_batch` to true - maybe I'm missing something @TimothyBJacobs